### ISSUE TITLE
convert dynamo exported onnx via pnnx

### ISSUE THREAD

**Issue by nihui** [Tencent/ncnn#5091]
- [ ] Abs
- [ ] Acos
- [ ] Acosh
- [ ] Add
- [ ] AffineGrid
- [ ] And
- [ ] ArgMax
- [ ] ArgMin
- [ ] Asin
- [ ] Asinh
- [ ] Atan
- [ ] Atanh
- [x] AveragePool
- [x] BatchNormalization
- [ ] Bernoulli
- [ ] BitShift
- [ ] BitwiseAnd
- [ ] BitwiseNot
- [ ] BitwiseOr
- [ ] BitwiseXor
- [ ] BlackmanWindow
- [ ] Cast
- [ ] CastLike
- [ ] Ceil
- [x] Celu
- [ ] CenterCropPad
- [x] Clip
- [ ] Col2Im
- [ ] Compress
- [ ] Concat
- [ ] ConcatFromSequence
- [ ] Constant
- [ ] ConstantOfShape
- [x] Conv
- [ ] ConvInteger
- [x] ConvTranspose
- [ ] Cos
- [ ] Cosh
- [ ] CumSum
- [ ] DFT
- [ ] DeformConv
- [ ] DepthToSpace
- [ ] DequantizeLinear
- [ ] Det
- [ ] Div
- [ ] Dropout
- [ ] DynamicQuantizeLinear
- [ ] Einsum
- [ ] Elu
- [ ] Equal
- [ ] Erf
- [ ] Exp
- [ ] Expand
- [ ] EyeLike
- [ ] Flatten
- [ ] Floor
- [x] GRU
- [ ] Gather
- [ ] GatherElements
- [ ] GatherND
- [x] Gelu
- [x] Gemm
- [ ] GlobalAveragePool
- [ ] GlobalLpPool
- [ ] GlobalMaxPool
- [ ] Greater
- [ ] GreaterOrEqual
- [ ] GridSample
- [ ] GroupNormalization
- [ ] HammingWindow
- [ ] HannWindow
- [x] HardSigmoid
- [x] HardSwish
- [ ] Hardmax
- [x] Identity
- [ ] If
- [ ] ImageDecoder
- [x] InstanceNormalization
- [ ] IsInf
- [ ] IsNaN
- [ ] LRN
- [x] LSTM
- [x] LayerNormalization
- [x] LeakyRelu
- [ ] Less
- [ ] LessOrEqual
- [ ] Log
- [x] LogSoftmax
- [ ] Loop
- [ ] LpNormalization
- [ ] LpPool
- [x] MatMul
- [ ] MatMulInteger
- [ ] Max
- [x] MaxPool
- [ ] MaxRoiPool
- [ ] MaxUnpool
- [ ] Mean
- [ ] MeanVarianceNormalization
- [ ] MelWeightMatrix
- [ ] Min
- [x] Mish
- [ ] Mod
- [ ] Mul
- [ ] Multinomial
- [ ] Neg
- [ ] NegativeLogLikelihoodLoss
- [ ] NonMaxSuppression
- [ ] NonZero
- [ ] Not
- [ ] OneHot
- [ ] Optional
- [ ] OptionalGetElement
- [ ] OptionalHasElement
- [ ] Or
- [x] PRelu
- [x] Pad
- [ ] Pow
- [ ] QLinearConv
- [ ] QLinearMatMul
- [ ] QuantizeLinear
- [x] RNN
- [ ] RandomNormal
- [ ] RandomNormalLike
- [ ] RandomUniform
- [ ] RandomUniformLike
- [ ] Range
- [ ] Reciprocal
- [ ] ReduceL1
- [ ] ReduceL2
- [ ] ReduceLogSum
- [ ] ReduceLogSumExp
- [x] ReduceMax
- [x] ReduceMean
- [x] ReduceMin
- [x] ReduceProd
- [x] ReduceSum
- [ ] ReduceSumSquare
- [ ] RegexFullMatch
- [x] Relu
- [ ] Reshape
- [x] Resize
- [ ] ReverseSequence
- [ ] RoiAlign
- [ ] Round
- [ ] STFT
- [ ] Scan
- [ ] Scatter
- [ ] ScatterElements
- [ ] ScatterND
- [x] Selu
- [ ] SequenceAt
- [ ] SequenceConstruct
- [ ] SequenceEmpty
- [ ] SequenceErase
- [ ] SequenceInsert
- [ ] SequenceLength
- [ ] SequenceMap
- [ ] Shape
- [ ] Shrink
- [x] Sigmoid
- [ ] Sign
- [ ] Sin
- [ ] Sinh
- [ ] Size
- [ ] Slice
- [x] Softmax
- [ ] SoftmaxCrossEntropyLoss
- [x] Softplus
- [x] Softsign
- [ ] SpaceToDepth
- [ ] Split
- [ ] SplitToSequence
- [ ] Sqrt
- [ ] Squeeze
- [ ] StringConcat
- [ ] StringNormalizer
- [ ] StringSplit
- [ ] Sub
- [ ] Sum
- [ ] Tan
- [x] Tanh
- [ ] TfIdfVectorizer
- [ ] ThresholdedRelu
- [ ] Tile
- [ ] TopK
- [ ] Transpose
- [ ] Trilu
- [ ] Unique
- [ ] Unsqueeze
- [x] Upsample
- [ ] Where
- [ ] Xor

**Comment by nihui** [comment]
```
95% tests passed, 24 tests failed out of 527

Total Test time (real) = 312.99 sec

The following tests FAILED:
         86 - test_nn_ChannelShuffle (Failed)
         91 - test_nn_Conv2d (Failed)
        105 - test_nn_GRU (Failed)
        111 - test_nn_InstanceNorm1d (Failed)
        112 - test_nn_InstanceNorm2d (Failed)
        113 - test_nn_InstanceNorm3d (Failed)
        122 - test_nn_LSTM (Failed)
        126 - test_nn_MultiheadAttention (Failed)
        137 - test_nn_RNN (Failed)
        292 - test_vit_b_32 (Failed)
        313 - test_pnnx_fuse_pad_conv1d (Failed)
        314 - test_pnnx_fuse_pad_conv2d (Failed)
        327 - test_torchvision_DeformConv2d (Failed)
        328 - test_torchvision_RoIAlign (Failed)
        398 - test_ncnn_nn_ChannelShuffle (Failed)
        401 - test_ncnn_nn_ConstantPad3d (Failed)
        417 - test_ncnn_nn_GRU (Failed)
        429 - test_ncnn_nn_LSTM (Failed)
        433 - test_ncnn_nn_MultiheadAttention (Failed)
        443 - test_ncnn_nn_ReplicationPad3d (Failed)
        444 - test_ncnn_nn_RNN (Failed)
        449 - test_ncnn_nn_Softmax2d (Failed)
        520 - test_ncnn_vit_b_32 (Failed)
        527 - test_ncnn_torchvision_DeformConv2d (Failed)
```

**Comment by nihui** [comment]
- [ ] Abs
- [ ] Acos
- [ ] Acosh
- [ ] Add
- [ ] AffineGrid
- [ ] And
- [ ] ArgMax
- [ ] ArgMin
- [ ] Asin
- [ ] Asinh
- [ ] Atan
- [ ] Atanh
- [ ] AveragePool
- [ ] BatchNormalization
- [ ] Bernoulli
- [ ] BitShift
- [ ] BitwiseAnd
- [ ] BitwiseNot
- [ ] BitwiseOr
- [ ] BitwiseXor
- [ ] BlackmanWindow
- [ ] Cast
- [ ] CastLike
- [ ] Ceil
- [ ] Celu
- [ ] CenterCropPad
- [ ] Clip
- [ ] Col2Im
- [ ] Compress
- [ ] Concat
- [ ] ConcatFromSequence
- [ ] Constant
- [ ] ConstantOfShape
- [ ] Conv
- [ ] ConvInteger
- [ ] ConvTranspose
- [ ] Cos
- [ ] Cosh
- [ ] CumSum
- [ ] DFT
- [ ] DeformConv
- [ ] DepthToSpace
- [ ] DequantizeLinear
- [ ] Det
- [ ] Div
- [ ] Dropout
- [ ] DynamicQuantizeLinear
- [ ] Einsum
- [ ] Elu
- [ ] Equal
- [ ] Erf
- [ ] Exp
- [ ] Expand
- [ ] EyeLike
- [ ] Flatten
- [ ] Floor
- [ ] GRU
- [ ] Gather
- [ ] GatherElements
- [ ] GatherND
- [ ] Gelu
- [ ] Gemm
- [ ] GlobalAveragePool
- [ ] GlobalLpPool
- [ ] GlobalMaxPool
- [ ] Greater
- [ ] GreaterOrEqual
- [ ] GridSample
- [ ] GroupNormalization
- [ ] HammingWindow
- [ ] HannWindow
- [ ] HardSigmoid
- [ ] HardSwish
- [ ] Hardmax
- [ ] Identity
- [ ] If
- [ ] ImageDecoder
- [ ] InstanceNormalization
- [ ] IsInf
- [ ] IsNaN
- [ ] LRN
- [ ] LSTM
- [ ] LayerNormalization
- [ ] LeakyRelu
- [ ] Less
- [ ] LessOrEqual
- [ ] Log
- [ ] LogSoftmax
- [ ] Loop
- [ ] LpNormalization
- [ ] LpPool
- [ ] MatMul
- [ ] MatMulInteger
- [ ] Max
- [ ] MaxPool
- [ ] MaxRoiPool
- [ ] MaxUnpool
- [ ] Mean
- [ ] MeanVarianceNormalization
- [ ] MelWeightMatrix
- [ ] Min
- [ ] Mish
- [ ] Mod
- [ ] Mul
- [ ] Multinomial
- [ ] Neg
- [ ] NegativeLogLikelihoodLoss
- [ ] NonMaxSuppression
- [ ] NonZero
- [ ] Not
- [ ] OneHot
- [ ] Optional
- [ ] OptionalGetElement
- [ ] OptionalHasElement
- [ ] Or
- [ ] PRelu
- [ ] Pad
- [ ] Pow
- [ ] QLinearConv
- [ ] QLinearMatMul
- [ ] QuantizeLinear
- [ ] RNN
- [ ] RandomNormal
- [ ] RandomNormalLike
- [ ] RandomUniform
- [ ] RandomUniformLike
- [ ] Range
- [ ] Reciprocal
- [ ] ReduceL1
- [ ] ReduceL2
- [ ] ReduceLogSum
- [ ] ReduceLogSumExp
- [ ] ReduceMax
- [ ] ReduceMean
- [ ] ReduceMin
- [ ] ReduceProd
- [ ] ReduceSum
- [ ] ReduceSumSquare
- [ ] RegexFullMatch
- [ ] Relu
- [ ] Reshape
- [ ] Resize
- [ ] ReverseSequence
- [ ] RoiAlign
- [ ] Round
- [ ] STFT
- [ ] Scan
- [ ] Scatter
- [ ] ScatterElements
- [ ] ScatterND
- [ ] Selu
- [ ] SequenceAt
- [ ] SequenceConstruct
- [ ] SequenceEmpty
- [ ] SequenceErase
- [ ] SequenceInsert
- [ ] SequenceLength
- [ ] SequenceMap
- [ ] Shape
- [ ] Shrink
- [ ] Sigmoid
- [ ] Sign
- [ ] Sin
- [ ] Sinh
- [ ] Size
- [ ] Slice
- [ ] Softmax
- [ ] SoftmaxCrossEntropyLoss
- [ ] Softplus
- [ ] Softsign
- [ ] SpaceToDepth
- [ ] Split
- [ ] SplitToSequence
- [ ] Sqrt
- [ ] Squeeze
- [ ] StringConcat
- [ ] StringNormalizer
- [ ] StringSplit
- [ ] Sub
- [ ] Sum
- [ ] Tan
- [ ] Tanh
- [ ] TfIdfVectorizer
- [ ] ThresholdedRelu
- [ ] Tile
- [ ] TopK
- [ ] Transpose
- [ ] Trilu
- [ ] Unique
- [ ] Unsqueeze
- [ ] Upsample
- [ ] Where
- [ ] Xor
